src/sql/parts/modelComponents/table.css

```diff
@@ -6,4 +6,13 @@
 .center-align
 {
 	text-align: center;
+}
+
+.slick-headerrow-column {
+	padding-left:3px !important;
+}
+
+.slick-cell
```

Comment:
should these styles be scoped more narrowly to avoid impacting everywhere using slickgrid (such as query editor and edit data)?

Reply:
sure I moved it to a separate class